### DISCUSSION TITLE
https://forecast.weather.gov - Access Denied

### DISCUSSION THREAD

**Discussion by vbguyny** [weather-gov/api#401] [Q&A] [answered]
Earlier today I was able to access the URL 
https://forecast.weather.gov/MapClick.php?lat=40.8156&lon=-73.001 but now I get the error: 

Access Denied
You don't have permission to access "http://forecast.weather.gov/MapClick.php?" on this server.
Reference #18.aa112817.1619645455.4a19b9

Please advise

**Comment by scadergit** [comment, marked as answer]
Any error code that starts with 18 is because the rate limits were exceeded. I'm not allowed to provide the limits 🙄 but that is a lot of requests...is that expected? The block will immediate lift after the amount of requests are reduced.

**Reply by vbguyny** [reply]
Ok, that makes sense.
There was a bug in my web site that put it into an infinite loop which lock up my site and caused the access denied error.  I can access the site now.  I also implemented memory caching so that it won't request the same data too soon and having the site wait 10 seconds when trying to request a resource again if there was an error.
Thanks!
